cli-commands/etherlime-ide/etherlime-ide.js

```diff
@@ -0,0 +1,85 @@
+const fs = require('fs-extra');
+const path = require('path');
+const git = require('simple-git/promise')();
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const spawn = require('child_process').spawn;
+
+const ideRepoUrl = 'https://github.com/LimeChain/Solidity-IDE.git';
+const ideFolder = 'Solidity-IDE';
+const ideServerRun = 'Solidity-IDE/solc-server.js';
+
+let projectWorkingDir = process.cwd(); //save current working dir of the project
+
+const run = async (port) => {
+    let etherlimeRootDir = getRootDir();
+    await fetchIdeRepo(etherlimeRootDir);
+    await runIde(etherlimeRootDir, port)
+}
+
+// find etherlime root dir
+const getRootDir = () => {
+    let currentDir = path.parse(__dirname)
+    return path.dirname(currentDir.dir)
+}
+
+//clone IDE repo or pull new updates if it was already initialized
+const fetchIdeRepo = async (rootDir) => {
+
+    if(fs.existsSync(`${rootDir}/${ideFolder}`)) {
+        console.log("====== Updating IDE ======")
+        try {
+            changeCurrentWorkingDir(`${rootDir}/${ideFolder}`) //change working dir in order to pull the repo
+            await git.pull('origin', 'master')
+        } catch (e) {
+        }
+        
+        fs.removeSync('./node_modules');
+        await installIdeModules()
+        return
+    }
+
+    console.log('====== Initializing IDE ======')
+    await git.clone(ideRepoUrl, `${rootDir}/${ideFolder}`);
```

Comment:
One thing that I am not sure about, but I think that `/` symbol might give you problems on Windows as the separator of paths there is other. I believe there is a constant in the fs library for path separator. Please double check it